power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/newton_raphson_se_solver.hpp

```diff
@@ -238,94 +244,50 @@ template <bool sym> class NewtonRaphsonSESolver {
                 }
                 // fill block with voltage measurement, only diagonal
                 if (row == col) {
-                    add_voltage_measurements(block, rhs_block, measured_value, row);
+                    process_voltage_measurements(block, rhs_block, measured_value, row);
```

Comment:
Should `measured_value` not be called `measured_values`? I think it contains al measurements right? (p.s. that is not in this PR and was already in the function call `prepare_matrix_and_rhs()`